ci/checksizes.sh

```diff
@@ -39,7 +39,7 @@ case $FULLVER in
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=71000
     REVERSI_OPT_GZ_EXPECTEDSIZE=32000
     ;;
-  2.11.7)
+  2.11.8)
     REVERSI_PREOPT_EXPECTEDSIZE=535000
     REVERSI_OPT_EXPECTEDSIZE=124000
```

Comment:
We should check whether this improved. I hope so; I had contributed a PR to 2.11.8 that was supposed to remove the dependency of `Vector` on `System.out.println`.